package.json

```diff
@@ -36,7 +36,7 @@
   "author": "Felix Kling",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "@babel/parser": "7.0.0-beta.53",
+    "@babel/parser": "7.0.0",
```

Comment:
can you now please change this to `^7.0.0`

Reply:
Oops, will do!